packages/visualizations/src/components/Legend/types.ts

```diff
@@ -0,0 +1,42 @@
+import type { Color } from '../types';
+
+export const LEGEND_POSITIONS = {
+    bottom: 'bottom',
+    top: 'top',
+    left: 'left',
+    right: 'right',
+} as const;
+export type LegendPositions = typeof LEGEND_POSITIONS[keyof typeof LEGEND_POSITIONS];
```

Comment:
I don't know how much we want to be precise in our types, now when we use custom legend we are able to write `left` and `top` in position but it has no effect on our legend as `legend-left` and `legend-top` classnames are not defined. It depends on the future and I don't know if we should restrict it now or leave it as it is with a small comment maybe ?

Reply:
I don't know what we want indeed, but with different types imports from ChartJS (and maybe Maplibre) it just seemed easier and not that risky to accept all at this point.